draft-ietf-httpbis-cache-latest.xml

```diff
@@ -291,29 +290,35 @@
    caches always store and reuse particular responses.
 </t>
 <t>
-   Each <x:dfn>cache entry</x:dfn> consists of a cache key and one or more
-   HTTP responses corresponding to prior requests that used the same key. The
-   most common form of cache entry is a successful result of a retrieval
+   Caches store HTTP responses corresponding to prior requests that used the
+   same cache key. Most commonly, this is a successful result of a retrieval
```

Comment:
I find this more confusing than the original. How about if we delete the first sentence and move the rest of this paragraph below the discussion of cache key?

Reply:
done